zion/src/main/scala/edu/uci/ics/cloudberry/zion/model/schema/Schema.scala

```diff
@@ -113,13 +180,19 @@ trait Measurement {
 case class Schema(typeName: String,
                   dimension: Seq[Field],
                   measurement: Seq[Field],
-                  primaryKey: Seq[String],
-                  timeField: String
+                  primaryKey: Seq[Field],
+                  timeField: Option[TimeField]
```

Comment:
Currently yes 😄 . We require each dataset should have a `timefield`.  later on the roadmap, if data is streamed in the system, we would like to propose a `txnTime` ourself if the original dataset doesn't have it.